[WIP][DEPENDENCY][STRIPE][NPM] Mega dependency update

fredteumer opened the pull request:
# Overview

Massive updates to the frontend bumping various dependencies and squashing some UI bugs as a result.

ArtSabintsev commented:
<img width="942" alt="Screenshot 2025-06-13 at 12 07 39 AM" src="https://github.com/user-attachments/assets/9e09e07a-8886-45ae-8bad-d40fb46d6d89" />
Still says 100k instead of 150k when upgrading the environment

fredteumer commented:
> Sending a request doesn't show the result (or an error).

This is also an issue with the environment of the preview. `rpc.grove.town` doesn't work since the migration. 
Here it is working locally with `rpc.grove.city` configured.

![image](https://github.com/user-attachments/assets/2a4cb739-f280-43d2-9baa-1f767127cf5f)

fredteumer commented:
> The bell icon is missing.

Fixed in commit https://github.com/pokt-foundation/grove-portal/pull/707/commits/c33944ed839cfe0b55b8fb0fa6331e7c4e63dab7 

> Still says 100k instead of 150k when attempting to upgrade to unlimited

Fixed in commit https://github.com/pokt-foundation/grove-portal/pull/707/commits/c33944ed839cfe0b55b8fb0fa6331e7c4e63dab7
